README.md

```diff
@@ -48,26 +48,26 @@ To install the SAP Cloud SDK test-util as development dependencies in your proje
 $ npm install -D @sap-cloud-sdk/test-util
 ```
 
-## API documentation
-We use GitHub Pages for our API documentation, the link can be found [here](https://sap.github.io/cloud-sdk/docs/js/api-reference-js-ts).
+## Documentation
+Check out our [API documentation](https://sap.github.io/cloud-sdk/docs/js/api-reference-js-ts)
+and [getting started](https://sap.github.io/cloud-sdk/docs/js/getting-started) guide on [documentation portal](https://sap.github.io/cloud-sdk/)
 
 ## How to switch to the Open Source version of the SAP Cloud SDK
 Please ignore this section, if you have never used the SAP Cloud SDK with a version `< 1.18.0`.
-If you are using an old version of the SAP Cloud SDK, you might want to read [how to migrate to this Open Source version of the SAP Cloud SDK](./knowledge-base/how-tos/how-to-switch-to-os-cloud-sdk.md).
+If you are using an old version of the SAP Cloud SDK, you might want to read [how to migrate to this Open Source version of the SAP Cloud SDK](https://sap.github.io/cloud-sdk/docs/js/guides/migrate-to-open-source-version-of-cloud-sdk-for-javascript-typescript).
 
 ## Related Projects
 
 ### SAP Cloud SDK CLI
 To reduce the development effort even more, you can use the [SAP Cloud SDK command line interface (CLI)](https://github.com/sap/cloud-sdk-cli) to start a new [NestJS](https://github.com/nestjs/nest) project, including the SDK right from the get go and supporting you do deploy your project to SAP Cloud Platform.
 
 ### Virtual Data Model (VDM)
-In addition to the Open Source parts of this project, we also publish the SAP Cloud SDK Virtual Data Model (VDM) - JavaScript / TypeScript clients for the latest versions of the whitelisted OData services of SAP S/4HANA Cloud under the SAP Developer licence. Those are not included in this project, but rely on modules from this project.
+In addition to the Open Source parts of this project, we also publish the SAP Cloud SDK Virtual Data Model (VDM) - JavaScript / TypeScript pre-generated type-safe clients for the latest versions of the whitelisted OData services of SAP S/4HANA Cloud under the SAP Developer license. Those are not included in this project, but rely on modules from this project.
```

Comment:
Good catch on the typo. I find it stumbly to read with the additional adjectives and I think it does not give you more relevant information. Do we have a general section on the VDM in our documentation? I couldn't find it. That could make sense here.
```suggestion
In addition to the Open Source parts of this project, we also publish the SAP Cloud SDK Virtual Data Model (VDM) - JavaScript / TypeScript clients for the latest versions of the whitelisted OData services of SAP S/4HANA Cloud under the SAP Developer license. Those are not included in this project, but rely on modules from this project.
```